### ISSUE TITLE
crash: libavformat.so.58+0x1513ee

### ISSUE THREAD

**Issue by shuaiyu-wang** [bytedeco/javacv#1852]
Why is my program crashing?
**source:** 
`while ((grabframe = grabber.grab()) != null) {

                if (startTime == 0) {
                    startTime = System.currentTimeMillis();
                }
                videoTS = 1000 * (System.currentTimeMillis() - startTime);
                if (videoTS > recorder.getTimestamp()) {
                    recorder.setTimestamp(videoTS);
                }
                recorder.record(grabframe);

            }`

**his_error_pid.log:**
`#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007f1ba08f33ee, pid=8629, tid=0x00007f1a70def700
#
# JRE version: Java(TM) SE Runtime Environment (8.0_221-b11) (build 1.8.0_221-b11)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (25.221-b11 mixed mode linux-amd64 )
# Problematic frame:
# C  [libavformat.so.58+0x1513ee]
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#

---------------  T H R E A D  ---------------

Current thread (0x00007f1adc04d000):  JavaThread "Thread-2502" daemon [_thread_in_native, id=22018, stack(0x00007f1a70cef000,0x00007f1a70df0000)]

siginfo: si_signo: 11 (SIGSEGV), si_code: 1 (SEGV_MAPERR), si_addr: 0x0000000000000000

Registers:
RAX=0x0000000000000000, RBX=0x00007f19dc890dc0, RCX=0x0000000000000000, RDX=0x8000000000000001
RSP=0x00007f1a70dee610, RBP=0x8000000000000000, RSI=0x0000000000000000, RDI=0x000000000000001e
R8 =0x000000000000000f, R9 =0x0000000000000001, R10=0x000000000000001e, R11=0x8000000000000001
R12=0x00007f19dc09bea0, R13=0x0000000000000000, R14=0x00007f19dd14fac0, R15=0x0000000000000000
RIP=0x00007f1ba08f33ee, EFLAGS=0x0000000000010246, CSGSFS=0x0000000000000033, ERR=0x0000000000000006
  TRAPNO=0x000000000000000e

Stack: [0x00007f1a70cef000,0x00007f1a70df0000],  sp=0x00007f1a70dee610,  free space=1021k
Native frames: (J=compiled Java code, j=interpreted, Vv=VM code, C=native code)
C  [libavformat.so.58+0x1513ee]

Java frames: (J=compiled Java code, j=interpreted, Vv=VM code)
J 4357  org.bytedeco.javacpp.avformat.av_write_frame(Lorg/bytedeco/javacpp/avformat$AVFormatContext;Lorg/bytedeco/javacpp/avcodec$AVPacket;)I (0 bytes) @ 0x00007f23ddc703a3 [0x00007f23ddc70340+0x63]
J 4671 C2 org.bytedeco.javacv.FFmpegFrameRecorder.recordImage(IIIIII[Ljava/nio/Buffer;)Z (990 bytes) @ 0x00007f23ddea16a8 [0x00007f23ddea09c0+0xce8]
J 6456 C2 com.example.videoreceiver.netty.pusher.RtmpPusher.recordPush()V (703 bytes) @ 0x00007f23de2fd50c [0x00007f23de2fbea0+0x166c]
J 7808 C1 com.example.videoreceiver.netty.pusher.RtmpPusher.run()V (5 bytes) @ 0x00007f23de83c40c [0x00007f23de83c380+0x8c]
v  ~StubRoutines::call_stub
`

**Comment by saudet** [comment]
It might be a bug in FFmpeg that was fixed. Please try with the snapshots:
http://bytedeco.org/builds/

**Comment by saudet** [comment]
Actually, the cause of this issue is probably the same as issue #1858, so this should be fixed in the snapshots. If you're still having problems even with the latest snapshots though, please let me know!

**Comment by saudet** [comment]
Duplicate of #1858
